cypress/support/auth-provider-commands/auth0.ts

```diff
@@ -32,7 +32,7 @@ Cypress.Commands.add("loginToAuth0", (username: string, password: string) => {
     {
       validate: () => {
         // Validate presence of access token in localStorage.
-        cy.wrap(localStorage).invoke("getItem", "authAccessToken").should("exist");
+        cy.wrap(localStorage).invoke("getItem", "authState").should("exist");
```

Comment:
where is `localStorage` coming from in this example?

Reply:
never mind 🤦🏻‍♀️  This might be better so we aren't pulling from globals
```suggestion
cy.window().its('localStorage).invoke('getItem', 'authState).should('exist')
```